testsuite/features/secondary/minssh_bootstrap_api_proxy.feature

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2017-2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scope_salt_ssh
+@scope_onboarding
+@ssh_minion
+@proxy
+Feature: Register a salt-ssh system via API and Proxy
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Delete SSH minion system profile before API bootstrap test
+    Given I am on the Systems overview page of this "ssh_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    Then "ssh_minion" should not be registered
+
+  Scenario: Bootstrap a SLES SSH minion via API
+    Given I am logged in API as user "admin" and password "admin"
+    And I block connections from "server" on "ssh_minion"
```

Comment:
Personally, the block and flush of connections I would put it in a different scenario, to highlight it, as it's kind of important to explain that we don't have connection to the server, as part of the test scenario.

_Also, it might be interesting a description of the acceptance criteria just below the Feature title_

Reply:
As discussed: I have integrated this into the existing feature test with extra scenarios which are executed only if a proxy is part of the test